lib/parseMigrate.js

```diff
@@ -7,10 +7,12 @@ const { promisify } = require('util');
 const promisifyExec = promisify(exec);
 const promisifyWriteFile = promisify(fs.writeFile);
 
+const srcDir = __dirname.replace('/lib', '');
+
 module.exports = async filePath => {
   try {
     const { stdout: docgenOut } = await promisifyExec(
-      `react-docgen ${filePath} --pretty --resolver findAllComponentDefinitions`
+      `node ${srcDir}/node_modules/react-docgen/bin/react-docgen.js ${filePath} --pretty --resolver findAllComponentDefinitions`
```

Comment:
Consider npx instead then you can do

no react-docgen...

Same for other lines

Reply:
ooooooo, I like this idea

Reply:
I'll take a look and see how it fits in, thanks for the recommendation

Reply:
@maniator Works with NPX. I thought this was magic at first until i realized it was bundled with node 5.2.0 and up, so I threw in a requirement for that v at minimum in the package.json as well